Exercise_1/interactive_command_list

```diff
@@ -1,7 +1,7 @@
 ## For exercises 1-2 use the following:
 srun --partition=express --nodes=1 --cpus-per-task=2 --pty --time=00:30:00 /bin/bash
 
-module load anaconda3/2021.05
+module load anaconda3/2022.05
 conda create -n myenv python=3.9 -y
 conda activate myenv
```

Comment:
I get an error when I run this because it has that my environment is not setup for a `conda activate` statement. You might just want to run a `source activate` because it also matches the documentation in the RTD as well.